src/content/en/2021/accessibility.md

```diff
@@ -446,15 +446,13 @@ The `placeholder` attribute was introduced in HTML5 in 2014. Its intended use is
   sql_file="placeholder_but_no_label.sql"
 ) }}
 
-The improper use of a placeholder as a replacement for the `<label>` element is surprisingly prevalent. Roughly 58% of desktop and mobile websites in this year's survey used the placeholder attribute. Of those sites, nearly 3 out of 4 of them included the placeholder attribute and failed to include an programmatically associated `<label>` element.
+The improper use of a placeholder as a replacement for the `<label>` element is surprisingly prevalent. Roughly 58% of desktop and mobile websites in this year's survey used the `placeholder` attribute. Of those sites, nearly 65% of them included the `placeholder` attribute and failed to include a programmatically associated `<label>` element.
```

Comment:
Changed this to 65% as didn't feel that was close enough to 3 out of 4 (75%)

Reply:
agreed